package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stripe/connect-js",
-  "version": "3.3.3",
+  "version": "3.3.5",
```

Comment:
We do not bump the version on the same PR as making the changes - can we do this separately?

Reply:
Np! Updated!